sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/BlobTriggerAttribute.cs

```diff
@@ -66,5 +66,10 @@ public string BlobPath
         {
             get { return _blobPath; }
         }
+
+        /// <summary>
+        /// Returns a bool value that indicates whether EventGrid is used.
+        /// </summary>
+        public bool UseEventGrid { get; set; }
```

Comment:
Are we going to have different trigger kinds? 

Is it worth to make this an enum?

Reply:
+1.
Changefeed would be another strategy once is GA.
If we create enum then we should also name existing strategy/strategies and put there to give user list of possibilities.

Reply:
enum is added.